src/Card/README.md

```diff
@@ -441,7 +511,11 @@ When using horizontal variant Paragon provides additional component `Card.Body`
 
   return (
     <Card orientation={isSmall ? "vertical" : "horizontal"}>
-      <Card.ImageCap src="https://source.unsplash.com/360x200/?nature,flower" srcAlt="Card image"/>
+      <Card.ImageCap
+        skeletonHeight={isSmall && 140}
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` prop here?